map_claims_test.go

```diff
@@ -7,55 +7,52 @@ import (
 func TestVerifyAud(t *testing.T) {
 	var nilInterface interface{}
 	var nilListInterface []interface{}
-	var intListInterface interface{} = []int{1,2,3}
```

Comment:
The changes on this file are only from go fmt? if that's the case I'd revert them the there is no change in the scope of this PR

Reply:
Thanks for reviewing! I'll make the requested changes.

Reply:
I removed changes related to map_claims_test.go.

Reply:
I still see the file changed, could it be an editor setting changing it automatically?